serverless/installing_serverless/removing-openshift-serverless.adoc

```diff
@@ -24,23 +24,22 @@ include::modules/serverless-uninstalling-knative-eventing.adoc[leveloffset=+1]
 You can remove the {ServerlessOperatorName} from the host cluster by following the documentation on xref:../../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster[deleting Operators from a cluster].
 
 [id="deleting-serverless-CRDs"]
-== Deleting {ServerlessProductName} CRDs
+== Deleting {ServerlessProductName} custom resource definitions
 
-After uninstalling the {ServerlessProductName}, the Operator and API CRDs remain on the cluster.
-You can use the following procedure to remove the remaining CRDs.
+After uninstalling the {ServerlessProductName}, the Operator and API custom resource definitions remain on the cluster.
```

Comment:
Same here re: adding "(CRD)" abbreviation in first instance (and then just "CRD" thereafter).

Reply:
Oh, looking further, seems like expanding out the spelling out each time was a deliberate choice. CRD is OK to use after the first instance, though, if you do want.

Reply:
Cool, I just wasn't sure if it read OK since it's used a fair bit but I'll do that to make it consistent. Thanks!